tox.ini

```diff
@@ -30,7 +30,7 @@ deps =
     pre-commit
 commands =
     pre-commit install-hooks
-    pre-commit run {posargs:--color always --all-files --show-diff-on-failure}
+    pre-commit run --color always --all-files --show-diff-on-failure {posargs}
```

Comment:
Since the CI runs this with a trailing `--` tox was skipping the important `--all-files` and instead trying to run on only non-committed files of which there were none.

Reply:
We could leave it there to always be green 😆